ngraph/test/visitors/op/sqrt.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+TEST(attributes, sqrt_op)
+{
+    using namespace opset1;
+
+    NodeBuilder::get_ops().register_factory<Sqrt>();
```

Comment:
Good practice is to have independent test. Static variables and singletons are not best approach.

Reply:
This is a common step in all of the existing `attributes` tests.
So it can be refactored in a separate PR if you have an idea how to improve it.

Reply:
Unfortunately I don't have.